README.md

```diff
@@ -1,7 +1,9 @@
 # Brown Esports Discord bot
+
```

Comment:
Just for reference, what formatter are you using? I used Prettier, but didn't think to format the README. As long as it's consistent with Prettier, this should be fine.

Reply:
I use Prettier as well, and my understanding is that Prettier auto-formats [markdown files](https://prettier.io/blog/2017/11/07/1.8.0.html#markdown-support) by default.